src/main/types/JWT.types.ts

```diff
@@ -1,7 +1,8 @@
+import { EcdsaSignature } from 'did-jwt/lib/util';
 import type { DIDResolutionResult, VerificationMethod } from 'did-resolver';
 // import type {JWK} from "jose/types";
 
-export type Signer = (data: string | Uint8Array) => Promise<string>;
+export type Signer = (data: string | Uint8Array) => Promise<EcdsaSignature | string>;
```

Comment:
Guess we should add a comment that this is the signer type/interface conforming to the DID-JWT module